agendas/2024/sprint-kickoff/YYYY-MM-DD-Flow-Sprint-Kickoff-Template.md

```diff
@@ -0,0 +1,227 @@
+# Overview
+
+ ### Team Wins 🎉
+ 
+ * ABI.encode/decode() functional subset of EVM work was deployed to testnet to unblock Axelar integration
+
+- 
+
+### Mainnet Uptime - Last 14 days (DD/MM/YY to DD/MM/YY) - Vishal
+
+|                         | Target | Current Score | Error budget used |
+|:------------------------|:------:|:-------------:|:-----------------:|
+| Collection Finalization | 99.9%   |    100%       |       0%         |
+| Block Finalization      | 99.9%   |    100%       |       0%         |
+| Transaction Execution   | 99.9%   |    100%       |       0%         |
+| Block Sealing           | 99.9%   |    100%       |       0%         |
+| Access API Liveness     | 99.9%   |    100%       |       0%         |
+
+#### Incidents
+- 
+
+
+# Working Group Updates
+
+### **Cadence and Virtual Machine** Jan
+Cycle Objective(s): 
+
+**Done last sprint**
+
+- General description of completed items
+- List of Closed issues
+
+
+**This sprint**
+
+- General description of sprint goal
+- List of issues to be worked on
+
+
+**On Hold**
+
+- List of issues on hold
+
+
+**Active Epics**
+
+- Main epics that are being tackled, and currently active for this working group
+
+---
+
+### **Core Protocol** Jerome
+Cycle Objective(s): 
+
+* Continue with adding BFT mitigations to enable 10 permissionless ANs
+* Continue design of Dynamic Protocol 
+
+**Done last sprint**
+
+- General description of completed items
+- List of Closed issues
+
+
+**This sprint**
+
+- General description of sprint goal
+- List of issues to be worked on
+
+
+**On Hold**
+
+- List of issues on hold
+
+
+**Active Epics**
+
+- Main epics that are being tackled, and currently active for this working group
```

Comment:
```suggestion
- Reinforcing Flow’s commitment to full protocol autonomy and scalability
- Improve network performance
- Improve network availability 
- Simplify community contributions to core protocol and maintainability
- Improve network reliability and data availability for dApp developers
- Data-driven Prioritization and Scaling Engineering
```

Reply:
@AlexHentschel - am curious whether we want to refer more to KRs than to epics since not all projects have epics (external teams etc) and sometime epics are worded could be opaque and not as clear.

Reply:
from my perspective, the main goal is for the information to be easily digestible. Your suggestion of using KRs seems to facilitate that and the information it less cryptic. Therefore, I think this is a great suggestion.